README.md

```diff
@@ -25,8 +25,11 @@ To fetch dependencies and get cooking:
 1. `npm install`
 2. As part of the npm install process, the `postinstall` script will install the Bower dependencies for you.
 3. Copy `config/local.json.example` to `config/local.json`, and put your local info in there.
-4. `npm start`
-5. You're up and running! surf to <http://localhost:8080> :surfer:
+4. Run `./server/db/create_db.sh` to create the database, tables, and indexes
```

Comment:
yikes, that's...

![](http://media.giphy.com/media/eah2loce5T8VW/giphy.gif)

srsly, give it a shot with a fresh 9.3.5 install (I think it's still stable? and that's the latest RDS version), paste your results in here, we can try to debug further